pylintrc

```diff
@@ -34,6 +34,9 @@ load-plugins=
 # multiple time (only on the command line, not in the configuration file where
 # it should appear only once).
 disable=
+    missing-docstring,
```

Comment:
I agree with @sarina. It's useful to have pylint flag new functions that don't have docstrings.